src/blockdata/constants.rs

```diff
@@ -250,5 +277,54 @@ mod test {
         assert_eq!(format!("{:x}", gen.header.block_hash()),
                    "00000008819873e925422c1ff0f99f7cc9bbb232af63a077a480a3633bee1ef6".to_string());
     }
+
+    // The *_chain_hash tests are sanity/regression tests, they verify that the const byte array
+    // representing the genesis block is the same as that created by hashing the genesis block.
+    fn chain_hash_and_genesis_block(network: Network) -> (String, String) {
```

Comment:
A comment explaining why we stringly compare would be useful for future readers eager to clean it up.